src/sif/autocost.cc

```diff
@@ -431,6 +442,8 @@ Cost AutoCost::EdgeCost(const baldr::DirectedEdge* edge,
             surface_factor_ * kSurfaceFactor[static_cast<uint32_t>(edge->surface())];
   if (edge->toll()) {
     factor += toll_factor_;
+  } else if (edge->use() == Use::kAlley) {
```

Comment:
should be different if

Reply:
a toll should never exist on a alley, but I will change.